pkg/apis/migration/migration.go

```diff
@@ -162,6 +166,62 @@ func GetVolumeMigrationService(ctx context.Context, volumeManager *cnsvolume.Man
 			informer.Informer().Run(stopCh)
 			log.Debugf("Informer started for cnsvspherevolumemigrations")
 		}()
+		crdCleanupTicker := time.NewTicker(time.Duration(defaultCRDCleanupIntervalInMin * time.Minute))
+		defer crdCleanupTicker.Stop()
```

Comment:
Wouldn't this stop the ticker once GetVolumeMigrationService is initialized? How did you test that this CRD cleanup routine is performing the cleanup every 5 mins?

Reply:
Good catch! I think there was a problem here. Hence I had kept the PR as `Work-in-progress`. Lets discuss this in the PR: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/440
Right now, I would like to keep the scope of this PR minimal so that we address the PV(reclaim policy retain) deletion path